app/helpers/home_helper.rb

```diff
@@ -1,6 +1,7 @@
 module HomeHelper
   def placeholder(type)
     case type
+      # when sollte gleiche Einrückungstiefe wie das case haben
```

Comment:
I find that arguable when using `then`, but we just switched to newlines anyways.